docs/source/resources/contributing.md

```diff
@@ -214,6 +214,50 @@ All documentation is checked using [Vale](https://vale.sh/). In documentation th
 
 The spelling of a project name should use the casing of the project, for example [PyPI](https://pypi.org/) should always be spelled as `PyPI` and not `pypi` or `PYPI`. If needed new words can be added to the `ci/vale/styles/config/vocabularies/aiq/accept.txt` and `ci/vale/styles/config/vocabularies/aiq/reject.txt` files.
 
+### Path Checks
+
+All documentation and files which match certain criteria are checked using a custom path check script.
+
+Path checks are used to ensure:
+* all symbolic links are valid
+* all paths within files are relative paths
+* all paths within files are valid (they exist)
+
+#### Skipping regions of files
+
+The check can be quite aggressive and may detect false positives. If a path is detected as invalid, but is actually valid, you can add comment(s) to the file to skip the check.
+
+To skip the **entire file**, ensure `path-check-skip-file` is present near the top of the file. Use comments appropriate for the file type.
+
+To skip a **section of the file**, ensure `path-check-skip` is present on the line above the section and `path-check-skip-end` is present on the line below the section. Use comments appropriate for the file type.
```

Comment:
Mention that this usually goes in a comment. Give some examples for different file types.